kubernetes/travis/deploy.sh

```diff
@@ -0,0 +1,28 @@
+export DEPLOY_BRANCH=${DEPLOY_BRANCH:-development}
+
+export REPOSITORY="https://github.com/${TRAVIS_REPO_SLUG}.git"
+
+sudo rm -f /usr/bin/git-credential-gcloud.sh
+sudo rm -f /usr/bin/bq
+sudo rm -f /usr/bin/gsutil
+sudo rm -f /usr/bin/gcloud
+rm -rf node_modules
+
+curl https://sdk.cloud.google.com | bash;
+source ~/.bashrc
+gcloud components install kubectl
+
+gcloud config set compute/zone us-central1-b
+openssl aes-256-cbc -K $encrypted_324c873b2450_key -iv $encrypted_324c873b2450_iv -in ./kubernetes/travis/susi-telegrambot-e467bca1e540.json.enc -out susi-telegrambot-e467bca1e540.json -d
```

Comment:
Make sure you have added these keys in travis repo settings of main repo and not in your fork's travis settings. You will need admin rights for that. We faced this problem while deploying susi server.